components/brave_rewards/browser/event_log.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_EVENT_LOG_H_
+#define BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_EVENT_LOG_H_
+
+#include <string>
+
+namespace brave_rewards {
+
+struct EventLog {
```

Comment:
naming: Since this is an entry in an event log, maybe `EventLogEntry`?

Reply:
Not sure if `Entry` adds additional value